h5p-bildetema/src/components/SearchPage/SearchInput/SearchInput.tsx

```diff
@@ -27,20 +29,23 @@ const SearchInput = ({
 
   return (
     <div className={`${styles.searchInputWrapper}`}>
-      <input
-        ref={ref}
-        // TODO needs a label
-        id={styles.searchInput}
-        className={`${rtl ? styles.rtl : ""}`}
-        placeholder={placeholder ?? ""}
-        value={search}
-        autoComplete="off"
-        onChange={e => handleSearch(e.target.value)}
-        onKeyDown={e => e.key === "Enter" && handleEnter()}
-      />
-      <span className={styles.searchInputIcon}>
-        <SearchIcon />
-      </span>
+      <label htmlFor={styles.searchInput}>
+        <span className={styles.visuallyHidden}>{searchInputLabel}</span>
+        <input
+          ref={ref}
+          id={styles.searchInput}
+          type="search"
+          className={`${styles.searchInput} ${rtl ? styles.rtl : ""}`}
+          placeholder={placeholder ?? ""}
+          value={search}
```

Comment:
Add  aria-label={searchInputLabel} for accessibility

Reply:
The search input already got a `<label>` that it's nested within, so adding `aria-label` is not needed then